### ISSUE TITLE
phpdox - encountered a problem and has terminated.

### ISSUE THREAD

**Issue by jeanfinck** [theseer/phpdox#240]
Hello, I'm configuring a new Jenkins/Slave server and encountered this problem. I dont know if it really is a bug, but I'm sending it to you

```
  [phpdox] Oups... phpDox encountered a problem and has terminated!
  [phpdox] 
  [phpdox] It most likely means you've found a bug, so please file a report for this
  [phpdox] and paste the following details and the stacktrace (if given) along:
  [phpdox] 
  [phpdox] PHP Version: 5.6.9 (Linux)
  [phpdox] PHPDox Version: 0.8.1
  [phpdox] ErrorException: E_NOTICE 
  [phpdox] Location: phar:///var/www/html/workspace/money/build/tools/phpdox.phar/phpdox/generator/enricher/git/Git.php (Line 70)
  [phpdox] 
  [phpdox] Undefined variable: foo
  [phpdox] 
  [phpdox] #0 phar:///var/www/html/workspace/money/build/tools/phpdox.phar/phpdox/generator/Generator.php(176): TheSeer\phpDox\Generator\Enricher\Git->enrichStart()
  [phpdox] #1 phar:///var/www/html/workspace/money/build/tools/phpdox.phar/phpdox/generator/Generator.php(140): TheSeer\phpDox\Generator\Generator->handleEvent()
  [phpdox] #2 phar:///var/www/html/workspace/money/build/tools/phpdox.phar/phpdox/Application.php(241): TheSeer\phpDox\Generator\Generator->run()
  [phpdox] #3 phar:///var/www/html/workspace/money/build/tools/phpdox.phar/phpdox/CLI.php(165): TheSeer\phpDox\Application->runGenerator()
  [phpdox] #4 /var/www/html/workspace/money/build/tools/phpdox.phar(450): TheSeer\phpDox\CLI->run()
  [phpdox] 
  [phpdox] 
  [phpdox]
```

**Comment by jeanfinck** [comment]
When I comment "disable_functions" at the php.ini the result is different

```
;disable_functions = exec,passthru,shell_exec,system,proc_open,popen,curl_multi_exec,show_source,proc_close,pcntl_exec,eval,symlink,fsockopen,dl
```

```
   [phpdox] 
   [phpdox] 
   [phpdox] Oups... phpDox encountered a problem and has terminated!
   [phpdox] 
   [phpdox] It most likely means you've found a bug, so please file a report for this
   [phpdox] and paste the following details and the stacktrace (if given) along:
   [phpdox] 
   [phpdox] PHP Version: 5.6.9 (Linux)
   [phpdox] PHPDox Version: 0.8.1
   [phpdox] ErrorException: E_ERROR 
   [phpdox] Location: phar:///var/www/html/workspace/money/build/tools/phpdox.phar/phpdox/logger/ShellProgressLogger.php (Line 131)
   [phpdox] 
   [phpdox] Class 'PHP_Timer' not found
   [phpdox] 
   [phpdox] No stacktrace available
   [phpdox] 
   [phpdox] 
   [phpdox] Result: 255
```

**Comment by theseer** [comment]
Interesting behavior: Getting an error for a non defined variable when the function where it's set by reference is disabled. But it's a good pointer that I should verify if can actually execute external processes. Thanks for finding that ;)

The followup-problem is a duplicate for #238 and been fixed with 0.8.1.1

**Comment by theseer** [comment]
Check added with commit dd6e39f50090709e389b805290695533d00769ce
